var/spack/repos/builtin/packages/xsdk/package.py

```diff
@@ -71,6 +73,8 @@ class Xsdk(Package):
     depends_on('petsc@develop+trilinos+mpi+hypre+superlu-dist+metis+hdf5~mumps+double~int64',
                when='@develop')
 
+    depends_on('dealii@develop~python~doc~slepc~gmsh+petsc+mpi+trilinos~int64+hdf5~netcdf+metis', when='@develop')
```

Comment:
slepc is now added to xsdk - so will '~slepc' here cause a conflict?

Reply:
>  will '~slepc' here cause a conflict?

no, it just means this version of `deali` won't build with Slepc.

Reply:
ok - If I understand correctly - 'spack install xsdk@develop' will install slepc - but won't use it for 'dealii'. is this correct?

Reply:
>  is this correct?

correct.